docs/source/custom-graphs.rst

```diff
@@ -75,10 +75,10 @@ Dask schedulers differ in the following ways:
 
 1.  You specify the entire graph as a Python dict rather than using a
     specialized API
-2.  You get a variety of schedulers ranging from single machine, single core 
-    to threaded, multiprocessing, distributed, and
-3.  The Dask single-machine schedulers have logic to execute the graph in a
-    way that minimizes memory footprint
+2.  You get a variety of schedulers, ranging from a single-machine, single-core
+    scheduler to threaded, multi-process, and distributed options
+3.  You benefit from logic to execute the graph in a way that minimizes memory
```

Comment:
Rephrased this so that the list has parallel structure. Also, added periods at end as per https://www.purchase.edu/editorial-style-guide/general-style-preferences/punctuation/bulleted-and-numbered-lists/.